icevision/models/checkpoint.py

```diff
@@ -208,8 +205,11 @@ def model_from_checkpoint(
 
     model_type = None
     if model_name:
-        lib, mod = model_name.split(".")
```

Comment:
I see `lib` was instantiated before but it's not instantiated anymore in your PR.
Was it used somewhere else?
Any reason behind its removal?

Reply:
It's no longer a binary splits so i do not believe they are needed any longer